sled-agent/src/services.rs

```diff
@@ -270,15 +267,20 @@ impl Config {
     }
 }
 
-// The filename of the ledger, within the provided directory.
-const SERVICES_LEDGER_FILENAME: &str = "services.json";
-
-// A wrapper around `ZoneRequest`, which allows it to be serialized
-// to a JSON file.
+/// A wrapper around `ZoneRequest`, which allows it to be serialized
+/// to a JSON file.
+// TODO: Should this be here? Could be in a different file
```

Comment:
TODO(me): clean this up.

This is the "ledger" of requests that is stored on M.2s on the Sled. Fortunately, it didn't change, because I *already* snuck a generation number in here.

Unfortunately, it got hoisted up into the "sled_agent.rs", which means that "services.rs" is slightly-less-aware of it. Probably shouldn't live here.